config/mappings/assessment_mapping_fund_round.py

```diff
@@ -0,0 +1,47 @@
+from config.mappings.assessment_mapping import (
+    cof_r2w2_assessment_mapping as cof_r2w2,
+)
+from fsd_utils import CommonConfig
+
+fund_round_to_assessment_mapping = {
+    f"{CommonConfig.COF_FUND_ID}:{CommonConfig.COF_ROUND_2_ID}": cof_r2w2,
+}
+
+
+def transform_to_assessor_task_list_metadata(
```

Comment:
Eventually this function will be making calls to the databse as well as config, so I wonder if this might be better placed in the route controller for returning assessment metadata